pkg/controller/cassandracluster/generator_test.go

```diff
@@ -499,6 +499,18 @@ func checkVarEnv(t *testing.T, containers []v1.Container, cc *api.CassandraClust
 				assert.Contains(envVar, env.Name)
 				assert.Equal(envVar[env.Name], env.Value)
 			}
+		} else {
+			// Check cassandra container env vars
+			podIP := v1.EnvVar{
+				Name: "POD_IP",
+				ValueFrom: &v1.EnvVarSource{
+					FieldRef: &v1.ObjectFieldSelector{
+						APIVersion: "v1",
+						FieldPath:  "status.podIP",
+					},
+				},
+			}
+			assert.Contains(container.Env, podIP)
```

Comment:
You're checking that cassandra container has the POD_IP env, but what about the common env vars between cassandra and bootstrap ?

Reply:
The only other env vars in the cassandra container are the jolokia-related ones: JOLOKIA_USER, JOLOKIA_PASSWORD, and CASSANDRA_AUTH_JOLOKIA.

As I mentioned above, I'd love to test for the presence for these, but that requires setting a `jolokia-auth` secret so this `cassandracluster` configuration works:
```
  imageJolokiaSecret:
    name: jolokia-auth
```
I'm not sure how to set a secret in the unit tests. Help?

Reply:
@toffer no need it's already used in e2e tests see https://github.com/Orange-OpenSource/casskop/blob/master/test/e2e/testdata/cassandracluster-1DC.yaml#L19-L20

Reply:
@cscetbon I see. These e2e tests can't pass if the readiness-probe.sh scripts fail, and the readiness-probe.sh script can't pass if the JOLOKIA env vars aren't set in the cassandra container.

Sounds good to me!